lab_1/stop_words.txt

```diff
@@ -541,4 +542,4 @@ your
 yours
 yourself
 yourselves
-zero
```

Comment:
Why replace with the same?

Reply:
I just deleted the last empty line, but if its forbidden I can realise it in the code.

Reply:
Better not to touch given files. Consider them as untouchables.